Makefile

```
@@ -79,7 +78,7 @@ endif
 include common/Makefile.common.mk
 
 # Default bundle image tag
-BUNDLE_IMG ?= ibm-secretshare-operator-bundle:$(OPERATOR_VERSION)
+BUNDLE_IMG ?= ibm-secretshare-operator-bundle:$(VERSION)
```

`RELEASE_VERSION` should replace `OPERATOR_VERSION`
```suggestion
BUNDLE_IMG ?= ibm-secretshare-operator-bundle:$(RELEASE_VERSION)
```